lib/clibuddy/main.rb

```diff
@@ -66,6 +66,7 @@ def cli_buddy_error(e)
       t << [p.bold("Error Code "), e.id]
       t << [p.bold("File "), @descriptor_file]
       t << [p.bold("Message "), e.message]
+      t << [p.bold("Cause "), e.cause] if e.cause
```

Comment:
What kind of info will this usually be?

Reply:
I only saw it being used for the ParseError messages and those generally were very helpful. But we could always restrict this later if we see it being not useful.